config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```
@@ -1,4 +1,32 @@
 presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-gci-gce-ipvs
+    always_run: false
+    optional: true
+    skip_report: false
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - args:
+        - --timeout=170
+        - --bare
+        - --scenario=kubernetes_e2e
+        - --
+        - --check-leaked-resources
+        - --env=KUBE_PROXY_MODE=ipvs
+        - --extract=ci/latest
+        - --gcp-master-image=gci
+        - --gcp-node-image=ubuntu
+        - --gcp-zone=us-west1-b
+        - --ginkgo-parallel=30
+        - --provider=gce
+        - --test_args=--ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
```

You're missing a flag
```suggestion
        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gci-gce-ipvs
        - --test_args=--ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
```

e.g. for the pull-kubernetes-e2e-gce job https://github.com/kubernetes/test-infra/blob/cf4eb1c6d2060675dbf72d04afdc13276ea9f070/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L55